lib/elastic_record/index/documents.rb

```diff
@@ -155,6 +155,16 @@ def search(elastic_query, options = {})
         get url, elastic_query.update('_source' => load_from_source)
       end
 
+      def multi_search(elastic_queries)
+        url = "_msearch"
+
+        queries = elastic_queries.flat_map do |query|
```

Comment:
This is more like "headers_and_bodies" or "headers_and_searches". Inside each "search" is a query and aggregations.